core/vm/actor/builtin/shared/shared.hpp

```diff
@@ -25,8 +24,9 @@ namespace fc::vm::actor::builtin {
    * @param miner
    * @return
    */
-  fc::outcome::result<GetControlAddressesReturn> requestMinerControlAddress(
-      Runtime &runtime, const Address &miner);
+  auto requestMinerControlAddress(Runtime &runtime, const Address &miner) {
```

Comment:
In my opinion this change reduces readability since return type is not explicit.
> use type deduction only to make the code clearer or safer, and do not use it merely to avoid the inconvenience of writing an explicit type
https://google.github.io/styleguide/cppguide.html#Type_deduction